miden-lib/src/transaction/inputs.rs

```diff
@@ -331,6 +341,7 @@ fn add_input_notes_to_advice_inputs(
                 );
             },
             InputNote::Unauthenticated { .. } => {
+                // NOTE: keep in sync with the `prologue::process_input_note` kernel procedure
                 note_data.push(Felt::ZERO);
```

Comment:
I would also mention that this is the `is_authenticated` flag.

Reply:
added